packages/api/src/external/carequality/document/process-outbound-document-query-resps.ts

```diff
@@ -200,6 +201,7 @@ function buildInterrupt({
     await setDocQueryProgress({
       patient: { id: patientId, cxId: cxId },
       downloadProgress: { status: "failed" },
+      convertProgress: { status: "failed" },
       requestId,
```

Comment:
Since we are setting convert as processing since the beginning we need to make sure that if its finished before either complete or failed that we also apply this to the convert status

Reply:
If not it will be left processing